### ISSUE TITLE
Enhancement: Store configuration in a dotfile

### ISSUE THREAD

**Issue by Fmstrat** [glowing-bear/glowing-bear#1123]
To leverage the ability to save settings in a dotfile across multiple systems (for when reinstalling machines), I would like to request all configurable settings (minus logins) be stored in a dotfile instead of in the browser cache when running in electron.

For instance, a JSON file of `~/.config/glowing-bear/settings.json` would be perfect.

**Comment by lorenzhs** [comment]
That would indeed be a nice improvement. Settings management is a bit annoying and we had enough problems with the current localstorage implementation, so adding a second settings backend that needs to be maintained doesn't sound very enticing. I don't think it'll happen unless someone who wants to see it implemented does so (and writes the tests to confirm that the two implementations behave the same way).

**Comment by baldurmen** [comment]
I agree this would be nice! I'm now using the Debian package on Debian Bullseye and it would be great to offer the option of using a file like `/etc/glowing-bear/settings.json` to configure the instance :)
